src/views/catalog/templatescatalog/components/TemplatesCatalogDrawer/hooks/useCreateDrawerForm.tsx

```diff
@@ -98,6 +102,9 @@ const useCreateDrawerForm = (
         vmObject.spec.template.spec.domain.cpu.cores = cpu?.cores;
         vmObject.spec.template.spec.domain.memory.guest = memory;
 
+        if (!vmObject.spec.template.metadata.labels) vmObject.spec.template.metadata.labels = {};
```

Comment:
We have a couple of `getLabels` functions in different places. Could you unify them into one in src/utils and use it here?

Reply:
yes, we can do it and should do it but not on this pr, I'm trying to keep it minimized as possible.. can u open a ticket for it?